srtcore/core.cpp

```diff
@@ -3093,9 +3087,7 @@ void CUDT::startConnect(const sockaddr *serv_addr, int32_t forced_isn)
         throw CUDTException(MJ_NOTSUP, MN_ISCONNECTED, 0);
 
     // record peer/server address
-    delete m_pPeerAddr;
-    m_pPeerAddr = (AF_INET == m_iIPversion) ? (sockaddr *)new sockaddr_in : (sockaddr *)new sockaddr_in6;
-    memcpy(m_pPeerAddr, serv_addr, (AF_INET == m_iIPversion) ? sizeof(sockaddr_in) : sizeof(sockaddr_in6));
+    m_PeerAddr = serv_addr;
```

Comment:
Not sure if it is needed, but `socketaddr_any` does not have the assignment operator `operator=`.

Reply:
The default copying operator= is used here. This class has also a trivial enough definition so that it doesn't need any elaborated definition of it.